Gemfile

```diff
@@ -0,0 +1,12 @@
+
+# Pull gem index from rubygems
+source 'https://rubygems.org'
+
+# Pin the version of bundle we support
+gem 'bundler', '2.2.29'
+
+group :test do
+  gem 'rspec-core', '~> 3.10.1'
+  gem 'rspec-collection_matchers', '~> 1.2.0'
+  gem 'rspec_junit_formatter'
+end
```

Comment:
Final newline is also missing here. VSCode and probably other editors insert this by default.